server/src/email-password/signup.ts

```diff
@@ -67,21 +71,30 @@ async function getUser(api: GraphQLClient, email: string): Promise<{ User }> {
   return api.request<{ User }>(query, variables)
 }
 
-async function createGraphcoolUser(api: GraphQLClient, email: string, password: string): Promise<string> {
+async function createGraphcoolUser(api: GraphQLClient, firstName: string, lastName:string, email: string, password: string, passwordConfirm: string, phoneNum:string): Promise<string> {
```

Comment:
I think we should structure the arguments such that there is just one of them per line; ensures that code-readability is improved.

Reply:
Seen you have done this already.Thank you.